worker/backend/cni_network.go

```diff
@@ -0,0 +1,242 @@
+package backend
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/containerd/containerd"
+	"github.com/containerd/go-cni"
+	"github.com/opencontainers/runtime-spec/specs-go"
+)
+
+//go:generate go run github.com/maxbrunsfeld/counterfeiter/v6 github.com/containerd/go-cni.CNI
+
+// CNINetworkConfig provides configuration for CNINetwork to override the
+// defaults.
+//
+type CNINetworkConfig struct {
+	// BridgeName is the name that the bridge set up in the current network
+	// namespace to connect the veth's to.
+	//
+	BridgeName string
+
+	// NetworkName is the virtual name used to identify the managed network.
+	//
+	NetworkName string
+
+	// Subnet is the subnet (in CIDR notation) which the veths should be
+	// added to.
+	//
+	Subnet string
+}
+
+// defaultCNINetworkConfig is the default configuration for the CNI network
+// created to put concourse containers into.
+//
+var defaultCNINetworkConfig = CNINetworkConfig{
+	BridgeName:  "concourse0",
+	NetworkName: "concourse",
+	Subnet:      "10.80.0.0/16",
```

Comment:
Do we expose a flag for this? I see this is the default, and there's a way to configure it (`WithCNINetworkConfig`), just wondering if we've bubbled that up, since we'll probably need to for e.g. CI integration tests.

Reply:
https://github.com/concourse/concourse/issues/5140